tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TspTool/TspTool.cs

```diff
@@ -11,25 +11,76 @@
 
 namespace Azure.Sdk.Tools.Cli.Tools
 {
+
+
     /// <summary>
     /// This tool provides functionality for initializing TypeSpec projects and converting existing Azure service swagger definitions to TypeSpec projects.
     /// Use this tool to onboard to TypeSpec for new services or convert existing services.
     /// </summary>
     [McpServerToolType, Description("Tools for initializing TypeSpec projects and converting existing Azure service swagger definitions to TypeSpec projects.")]
     public class TypeSpecTool(INpxHelper npxHelper, ILogger<TypeSpecTool> logger, IOutputService output) : MCPTool
     {
+        /// <summary>
```

Comment:
You will need to rebase with latest and move this class into TypeSpec namespace. I would also like to see this new tool added in its own MCPTool class. We are trying to get to a stage one tool per class to enforce some cli hierarchy and as part of that work, we will split current class with multiple tools into its own class per tool call.

Reply:
@jsquire will be happy to read this ;)

Reply:
Yay!